tests/strands/event_loop/test_event_loop.py

```diff
@@ -3,7 +3,6 @@
 from unittest.mock import MagicMock, call, patch
 
 import pytest
-
```

Comment:
Do we know why so many files were reformatted as part of this?  If the dependencies were the same I would expect no formatting changes